index.html

```diff
@@ -4,36 +4,173 @@
+                    <li>JavaScript</li>
+                </ul>
+                <div class="see-project-btn">
+                    <p>See this project <img src="/03_Resources/01_Graphics/02_Stock/right-white-arrow.svg" alt="Right white arrow" class="arrow-see-project-btn"></p>
+                </div>
+            </div>
+            <div class="projects-index project6-section">
+                <img src="/04_Mockups/project6-mockup.png" alt="Project 6 Mockup" class="project-mockup project6-mockup">
+                <h2 class="projectname projectname6">
+                    Project name goes here
+                </h2>
+                <ul class="projects-languages project-languages6">
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <div class="see-project-btn">
+                    <p>See this project <img src="/03_Resources/01_Graphics/02_Stock/right-white-arrow.svg" alt="Right white arrow" class="arrow-see-project-btn"></p>
+                </div>
+         </div>
+        </div>
+    </section>
+    <section class="wrapper-about">
+        <div class="about-text">
+            <h1 class="title-about">About</h1>
+            <p class="summary-about">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        </div>
+        <div class="cv-btn">
+            <p>Get my CV</p> <img src="03_Resources/01_Graphics/02_Stock/download-btn.svg" alt="Download botton" class="download-btn">
+        </div>
+    </section>   
+    <section class="wrapper-skills">
+            <div class="languages-about">
+                <h2 class="title-languages-about">Languages</h2>
+                <ul class="list-languages-about">
+                    <li>JavaScript(ES5/ES6)</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="frameworks-about">
+                <h2 class="title-frameworks-about">Frameworks</h2>
+                    <ul class="list-frameworks-about">
+                        <li>Bootstrap</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capibara</li>
+                        <li>Selenium</li>
+                    </ul>
+            </div> 
+            <div class="skills-about">
+                <h2 class="title-skills-about">Skills</h2>
+                    <ul class="list-skills-about">
+                        <li>Codekit</li>
+                        <li>Codepen</li>
+                        <li>Github</li>
+                        <li>GitLAB</li>
+                        <li>Terminal</li>
+                    </ul>
+            </div> 
+    </section>
```

Comment:
- [ ] It would be better to use `ul` and `li` for categories in the `About myself` section to improve structure and semantics. Languages, Frameworks, and Skills should be list items. kindly change `section` to `ul` and `div` to `li`.